packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/AssertionExtensions.kt

```diff
@@ -4,9 +4,23 @@
 
 package com.mongodb.jbplugin.fixtures
 
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.utils.keyboard
 import com.intellij.remoterobot.utils.waitFor
 import java.time.Duration
 
+/**
+ * Closes all open modals by using the ESC key. This should be enough for recovering on a lot of different test cases.
+ */
+fun RemoteRobot.closeAllOpenModals() {
+    // just do it a few times, to ensure we closed all modals
+    for (i in 0..5) {
+        keyboard {
+            escape()
+        }
+    }
+}
```

Comment:
nit: It might serve us better to rename this to `closeNOpenModals(n: Number)`?

Reply:
I wanted to keep the number hidden because the purpose is like "cleaning the modals". 5 is a magic number, hopefully it works for our use cases. Usually you won't have more than 1 or 2 modals open.

Reply:
but this should not worry us - its single place so we can surely fix easily when something is not working